### PR TITLE
Spruce up maps + cleanup

### PR DESCRIPTION
**Pull request by lindsaycarr**
One thing I can't figure out - stripe of base map, not covered by gridded precip even though the basemap was created based on that data's bbox.

image

# Here's where this PR brings us:

### overall storm transpose map

image

### inputs-data map

image

### inputs-geometry map

image

### clipped data map

image

### melted data map

image

### CODE REVIEW

**Comment by lindsaycarr** [comment]
This would close out #14, #25, #35, and #46

File: scripts/visualize/visualize.create_basemap.R [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  # create basemap
+  basemap_data <- ggmap::get_map(location = geom_bbox, color = "bw", maptype = "toner")
+  
+  # expand bbox if needed
+  # basemap_bbox <- attr(basemap_data, "bb")
+  # bottom <- ifelse(basemap_bbox[["ll.lat"]] > geom_bbox[2, 1], geom_bbox[2, 1], basemap_bbox[["ll.lat"]])
+  # top <- ifelse(basemap_bbox[["ur.lat"]] < geom_bbox[2, 2], geom_bbox[2, 2], basemap_bbox[["ur.lat"]])
+  # left <- ifelse(basemap_bbox[["ll.lon"]] > geom_bbox[1, 1], geom_bbox[1, 1], basemap_bbox[["ll.lon"]])
+  # right <- ifelse(basemap_bbox[["ur.lon"]] < geom_bbox[1, 2], geom_bbox[1, 2], basemap_bbox[["ur.lon"]])
+  # attr(basemap_data, "bb") <- data.frame(ll.lat = bottom, ll.lon = left, ur.lat = top, ur.lon = right)
```
Comment (wdwatkins) [inline scripts/visualize/visualize.create_basemap.R]: best to not commit commented code

**Comment by wdwatkins** [comment]
re: the missing stripe of data, my guess would be that maybe that stripe is just slightly narrower than the raster cell size, so it can't fit a row of data there?  Could be stemming from the GDP request, or something in the plotting maybe.

**Comment by wdwatkins** [comment]
Everything else looks good 👍

File: scripts/visualize/visualize.create_basemap.R (hunk 1 above)
Comment (lindsaycarr) [reply scripts/visualize/visualize.create_basemap.R]: whoops! forgot that last piece :)

**Comment by lindsaycarr** [comment]
Storm transpose fig is actually showing the difference in average precip when the watershed is transposed north, so it is using the scale. I wonder if there's a way to make that more obvious ... I should add the actual values on the map somewhere.

**Comment by wdwatkins** [comment]
Oh I see, just wasn't thinking of it in that context.  Probably fine as long as the text says that then.

**Comment by lindsaycarr** [comment]
@wdwatkins ??

image

**Comment by lindsaycarr** [comment]
yeah, I think I should just do that - don't really like it on the map, it's not as pretty

**Pull request merged**